components/script/dom/htmlanchorelement.rs

```diff
@@ -575,15 +571,14 @@ impl Activatable for HTMLAnchorElement {
             }
         }
 
-        // Step 4.
-        //TODO: Download the link is `download` attribute is set.
-
         // https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-delivery
         let referrer_policy = match self.RelList().Contains("noreferrer".into()) {
```

Comment:
We should remove this code and replace it by implementing the missing step 12 of `follow_hyperlink`. You can use the `determine_policy_for_token` helper function to get a ReferrerPolicy value from a string.